Dockerfile.x86

```diff
@@ -1,53 +1,57 @@
+#RUN echo "zchunk=False" >> /etc/dnf/dnf.conf && \
+#    dnf install -y git gcc httpd python3 python3-pip python3-devel \
+#        python3-mod_wsgi mod_auth_openidc libffi-devel openssl-devel
 
-#RUN yum -y install rh-python36
-#RUN yum -y install rh-hython36-pip 
+WORKDIR /app
 
-RUN yum -y install make gcc libffi-devel openssl-devel bzip2-devel wget wireshark git python36.x86_64 python36-devel.x86_64\
-  && curl https://bootstrap.pypa.io/get-pip.py -o get-pip.py \
-  && python3 get-pip.py
 
-RUN python3 --version
-# && pip3.4 install --upgrade pip \
-# && pip3.4 install 'setuptools>=18.5' \
-# && cd /app
- 
-RUN mkdir /app/openshift-acct-mgt
-#  && cd /app/openshift-acct-mgt \
-#  && pip3 install gunicorn \
-#  && pip3 install Flask \
-#  && pip3 install kubernetes \
-#  && pip3 install openshift
+RUN rpm --initdb \
+ && yum clean all \
+ && yum -y install make gcc libffi-devel openssl-devel bzip2-devel wget wireshark git python3 httpd httpd-devel httpd-tools  python3-devel python3-pip \ 
+ && pip3 install --upgrade pip \
+ && pip3 install setuptools \
+ && pip3 install mod_wsgi \
+ && pip3 install Flask-HTTPAuth \
+ && yum install -y cpan
 
 COPY wsgi.py /app/openshift-acct-mgt/wsgi.py
-copy openshift_user.py /app/openshift-acct-mgt/openshift_user.py
-copy openshift_identity.py /app/openshift-acct-mgt/openshift_identity.py
-copy openshift_rolebindings.py /app/openshift-acct-mgt/openshift_rolebindings.py
-copy openshift_project.py /app/openshift-acct-mgt/openshift_project.py
-copy openshift_role.py /app/openshift-acct-mgt/openshift_role.py
+COPY openshift_user.py /app/openshift-acct-mgt/openshift_user.py
+COPY openshift_identity.py /app/openshift-acct-mgt/openshift_identity.py
+COPY openshift_rolebindings.py /app/openshift-acct-mgt/openshift_rolebindings.py
+COPY openshift_project.py /app/openshift-acct-mgt/openshift_project.py
+COPY openshift_role.py /app/openshift-acct-mgt/openshift_role.py
+COPY openshift.py /app/openshift-acct-mgt/openshift.py
 
 COPY start.sh /app/openshift-acct-mgt/start.sh
 COPY requirements.txt /app/openshift-acct-mgt/requirements.txt
 COPY config.py /app/openshift-acct-mgt/config.py
 
 RUN cd /app/openshift-acct-mgt \
-  && pip3 install --upgrade pip \
-  && pip3 install -r requirements.txt
-
-RUN chmod -R 777 /app
-RUN chmod -R 777 /usr
-
-EXPOSE 8080 8443
+ && pip3 install -r requirements.txt \
+ && chmod -R 777 /app \
+ && chmod -R 777 /usr
+
+
+# Note(knikolla): This is required to support the random
+# user IDs that OpenShift enforces.
+# https://docs.openshift.com/enterprise/3.2/creating_images/guidelines.html
+RUN chmod -R g+rwX /opt && \
+    chgrp -R 0 /opt && \
+    chmod -R g+rwX /run/httpd && \
+    chgrp -R 0 /run/httpd && \
+    chmod -R g+rwX /etc/httpd/logs && \
+    chgrp -R 0 /etc/httpd/logs && \
+    chmod -R g+rwX /etc/httpd/conf.d && \
+    chgrp -R 0 /etc/httpd/conf.d && \
+    sed -i -e "s|Listen 80|Listen 8080|g;" /etc/httpd/conf/httpd.conf
+
+EXPOSE 8080 8443 
```

Comment:
This image doesn't run any services on port 8443.

Separately, the `EXPOSE` keyword isn't necessary, but I have no complaints if you want to keep it around for informational purposes.